CHANGELOG.md

```
@@ -1,5 +1,17 @@
 # Changelog
 
+## [1.1.0] - 2026-03-17
+
+### Added
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)